src/phoebus_utils/unit_conversions.hpp

```diff
@@ -58,6 +58,38 @@ class UnitConversions {
   Real temperature_;
 };
 
+// Object for holding physical constants in code units for convenience
+class CodeConstants {
+  using pc = parthenon::constants::PhysicalConstants<parthenon::constants::CGS>;
+
+ public:
+  CodeConstants(UnitConversions unit_conv) {
+    const Real TIME = unit_conv.GetTimeCGSToCode();
+    const Real MASS = unit_conv.GetMassCGSToCode();
+    const Real LENGTH = unit_conv.GetLengthCGSToCode();
+    const Real ENERGY = MASS * LENGTH * LENGTH / (TIME * TIME);
+    const Real TEMPERATURE = unit_conv.GetTemperatureCGSToCode();
+    h_code_ = pc::h * MASS * LENGTH * LENGTH / TIME;
+    c_code_ = pc::c * LENGTH / TIME;
+    kb_code_ = pc::kb * ENERGY / TEMPERATURE;
+    mp_code_ = pc::mp * MASS;
+  }
+
+  Real h() const { return h_code_; }
```

Comment:
constants in code units show up occasionally in phoebus radiation right now, e.g. going from frequency in code units to neutrino energy in code units.

Reply:
All this should be `KOKKOS_FORCEINLINE_FUNCTION`. Except the constructor.

Although actually, is there any reason this class can't be a struct? Just make all the code units vars public and get rid of the getter functions. It's a simple enough little thing that that makes sense to me.

Reply:
Doing that while keeping the vars `const` would require putting the whole constructor in the initializer list which in my opinion would be unreadable. Maybe theres another way?

Reply:
Oh nice, I didn't know you could call other constructors in initializer lists like in that second option. I like that a lot, I just implemented it, thanks :+1: